transitions/index.js

```diff
@@ -103,6 +117,83 @@ const processChange = (change, callback) => {
     });
 };
 
+const deleteInfoDoc = (change, callback) => {
+  db.medic.get(`${change.id}-info`, (err, doc) => {
+    if (err) {
+      if (err.statusCode === 404) {
+        // don't worry about deleting a non-existant doc
+        return callback();
+      }
+      return callback(err);
+    }
+    doc._deleted = true;
+    db.medic.insert(doc, callback);
+  });
+};
+
+const getAdminNames = callback => {
+  const options = { key: [ 'user-settings' ], include_docs: true };
+  db.medic.view('medic-client', 'doc_by_type', options, (err, results) => {
+    if (err) {
+      return callback(err);
+    }
+    const admins = results.rows
+      .filter(row => {
+        return row.doc.roles &&
+               (row.doc.roles.includes('_admin') ||
+               row.doc.roles.includes('national_admin'));
```

Comment:
I realise why we need to do this this way, but we really need a better way of determining whether or not someone is online-first or offline-first.

Can we add a TODO here and link that to a ticket to add this as a flag in the user document? Linking the TODO so we'd know to change this code later. Can discuss when you're online

Reply:
Created issue: https://github.com/medic/medic-webapp/issues/3960

I didn't add the TODO because the code here got moved into a view as per your previous comment.